### PR TITLE
Calculator

### LINKED ISSUE #29: Calculator
**Is your feature request related to a problem? Please describe.**

Its a new feature/project.

**Describe the solution you'd like**
I would like to add a  Javascript Calculator.

### PR DESCRIPTION
**Pull request by Baba-Yaga-1**
Resolved #29 

I have added the calculator. Can you check the PR ?

### CODE REVIEW

**Review by yashsehgal — CHANGES_REQUESTED** [review]
Hey @Baba-Yaga-1 👋🏽 
Your code changes look good to me. There are some minor fixes you need to do first, after that I will merge your PR 🚀 

- Firstly, Fetch all the changes from upstream as there are some updates in common modules.
- Then change the `id` in the JSON file from `7` to `8`
- Update your PR with these changes and I will merge it then :D

**Comment by Baba-Yaga-1** [comment]
I made the PR 2 days back , maybe that's why conflict happened. And yeah I will fix the issues.

**Comment by Baba-Yaga-1** [comment]
I have updated the json file. Can you check it now ?

File: projects/project.json [hunk 1]
```diff
@@ -1,51 +1,58 @@
...
+       "github_url": "https://github.com/WH0SUMIT/propack/tree/main/projects/projectList/color-guessor"
+   }, {
+       "id": "7",
+       "title" : "Propack Form Validation in javascript",
+       "description" : "A basic Javascript form validation site.",
+       "live" : "./projectList/form-validation/index.html",
+       "github_url": "https://github.com/WH0SUMIT/propack/tree/main/projects/projectList/form-validation"
+   },
+   {
+    "id": "7",
```
Comment (yashsehgal) [inline projects/project.json]: @Baba-Yaga-1 Please do change the project ID to `8`

Like `"id": "8"`

**Review by yashsehgal — CHANGES_REQUESTED** [review]
Everything is fine now, Just need to change that ID thing. Please look into it @Baba-Yaga-1

**Comment by Baba-Yaga-1** [comment]
I updated the id that day itself, so now its showing nothing to commit.

example1

**Comment by yashsehgal** [comment]
> I updated the id that day itself, so now its showing nothing to commit. What should I do ?
> 
> example1

Alright, @Baba-Yaga-1 👍🏽 I will merge this as of now, If there will be any such bugs, I will resolve them by myself 👍🏽 

**Good work 💯**

**Pull request merged**